types/atom-keymap/README.md

```diff
@@ -19,3 +19,19 @@ Many of the types used by Atom Keymap can be referenced from the AtomKeymap name
 ```ts
 function example(keybind: AtomKeymap.KeyBinding) {}
 ```
+
+### Exposing Private Methods and Properties
```

Comment:
We won't actually publish a custom README, maybe this should be moved to `index.d.ts`?

Reply:
This was modeled after the ```@types/angular``` definition. I'm aware that it wont be published to NPM, but there is a link from NPM directly to the definition page within the repository. I don't think it's the kind of thing that makes it directly into the definition file, especially not without markup.

I'll remove it if this isn't something you're interested in having within the repository.